runtime/doc/treesitter.txt

```diff
@@ -382,6 +382,19 @@ get_captures_at_position({bufnr}, {row}, {col})
     Return: ~
         (table) A table of captures
 
+get_node_at_cursor({winnr}, {opts})                     *get_node_at_cursor()*
+    Gets the smallest named node under the cursor
```

Comment:
why does this depend on the cursor instead of just taking a position arg?

Reply:
Yes, that is going to be refactored, see my last comment.